.github/workflows/check-submodule-pointers.yml

```
@@ -0,0 +1,25 @@
+name: Check submodule pointers
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check-submodules:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Compare submodule pointers to lgtm.com branch
+        env: 
+          GITHUB_TOKEN: ${{ github.token }}
+        run: |
+            if [ $(git rev-parse @:./ql) != $(gh api repos/github/codeql/git/ref/heads/lgtm.com --jq '.object.sha') ] || \
+               [ $(git rev-parse @:./codeql-go) != $(gh api repos/github/codeql-go/git/ref/heads/lgtm.com --jq '.object.sha') ]
```

Be sure to always quote in bash:

```suggestion
            if [ "$(git rev-parse @:./ql)" != "$(gh api repos/github/codeql/git/ref/heads/lgtm.com --jq '.object.sha')" ] || \
               [ "$(git rev-parse @:./codeql-go)" != "$(gh api repos/github/codeql-go/git/ref/heads/lgtm.com --jq '.object.sha')" ]
```